### PR TITLE
Package, container for snapp-test-transaction

### LINKED ISSUE #9944: Snapp test transaction tool in CI
deb + docker image for devnet profile

### PR DESCRIPTION
**Pull request by psteckler**
In CI, build a deb package `mina-snapp-test-txn`, and a Docker container that installs the package.

As for the other Docker containers, there are `stretch` and `buster` versions. Tested by pulling the images to localhost, and running:
```
docker run -it the-image-sha -- mina-snapp-test-transaction
```
Both images ran the app successfully.

Closes #9944.

### CODE REVIEW

File: dockerfiles/Dockerfile-snapp-test-txn [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        libssl1.1 \
+        libpq-dev \
+        apt-transport-https \
+        ca-certificates \
+        dnsutils \
+	apt-utils \
+        man \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN mkdir /healthcheck && curl https://raw.githubusercontent.com/MinaProtocol/mina/develop/dockerfiles/scripts/healthcheck-utilities.sh -o /healthcheck/utilities.sh
```
Comment (mrmr1993) [inline dockerfiles/Dockerfile-snapp-test-txn:33]: Should the `develop` branch here be a variable?

File: scripts/rebuild-deb.sh [hunk 2]
```diff
@@ -82,6 +82,46 @@ ls -lh mina*.deb
...
+
+mkdir -p "${BUILDDIR}/DEBIAN"
+cat << EOF > "${BUILDDIR}/DEBIAN/control"
+
+Package: mina-snapp-test-txn
+Version: ${MINA_DEB_VERSION}
+License: Apache-2.0
+Vendor: none
+Architecture: amd64
+Maintainer: o(1)Labs <build@o1labs.org>
```
Comment (mrmr1993) [inline scripts/rebuild-deb.sh:95]: Nit: `O(1) Labs`

File: scripts/rebuild-deb.sh [hunk 3]
```diff
@@ -82,6 +82,46 @@ ls -lh mina*.deb
...
+ Built from ${GITHASH} by ${BUILD_URL}
+EOF
+
+echo "------------------------------------------------------------"
+echo "Control File:"
+cat "${BUILDDIR}/DEBIAN/control"
+
+# Binaries
+mkdir -p "${BUILDDIR}/usr/local/bin"
+cp ./default/src/app/snapp_test_transaction/snapp_test_transaction.exe "${BUILDDIR}/usr/local/bin/mina-snapp-test-transaction"
```
Comment (mrmr1993) [inline scripts/rebuild-deb.sh:111]: Intuitively it feels to me that the name of the installed binary should match the name of the package.

Comment (psteckler) [reply scripts/rebuild-deb.sh:111]: I'm going to change the package name to `mina-snapp-test-transaction`.

File: dockerfiles/Dockerfile-snapp-test-txn (hunk 1 above)
Comment (psteckler) [reply dockerfiles/Dockerfile-snapp-test-txn:33]: consulting with Andrew on that one

Comment (psteckler) [reply dockerfiles/Dockerfile-snapp-test-txn]: removed that line

**Review by mrmr1993 — APPROVED** [review]

**Review by bkase — APPROVED** [review]
Looks good to me

**Pull request merged**